### ISSUE TITLE
Error: <ellipse> attribute rx: Expected length, "NaN". when using url from `URL.createObjectURL`

### ISSUE THREAD

**Issue by tochibedford** [katspaugh/wavesurfer-multitrack#33]
Hey so I'm using this plugin for wavesurfer in React and for some reason I get errors when trying to load audio from an existing `File` Object:
![image](https://github.com/katspaugh/wavesurfer-multitrack/assets/34871260/7fe4f87b-0c52-45de-964a-06a513bcdb56)

Loading the audio directly from a public url works perfectly as shown in the Multitrack example though

What I've tried:
- I have tried using the URL.createObjectURL directly to create a url for the file object
- I have also tried converting the contents of the file to a base64String and passing that to multitrack.addTrack, but I keep getting the same error
![image](https://github.com/katspaugh/wavesurfer-multitrack/assets/34871260/2214a08f-3216-47e7-9c42-43ed4b439b16)

**Comment by katspaugh** [comment]
Strange, both should work fine. Check that your data-URL starts with `data:audio/mp3;base64,` and not just a bare base64 string.

**Comment by katspaugh** [comment]
Yes, unfortunately you cannot add new tracks without having a placeholder first. See #22.

I'm open to pull requests to refactor that, shouldn't be difficult.
